.github/workflows/test.yml

```diff
@@ -13,15 +13,16 @@ jobs:
       matrix:
         os: [macos-latest, windows-2019]
         node: [16.x, 18.x, 20.x, 22.x]
+      fail-fast: false
```

Comment:
Not necessary and happy to revert - but without fail-fast, when one task fails, all are canceled.  That isn't what we want from a CI testing matrix.